### PR TITLE
Content proposal in C# (fr) and Spring (en)

### PR DESCRIPTION
**Pull request by C-Alexis4414**
## What does this PR do?
Added C# and Spring learning resources.

### Description
For C# courses, it's a playlist on the Pentiminax channel.
And for the books on Spring, there are several PDFs on the Baeldung site.

### Why is this valuable (or not)?
It is useful for learning object-oriented programming in a dotNET environment or in Java with the Spring framework.

### How do we know it's really free?
Yes it's free.
For PDFs on Spring, you must enter your email address to receive them.

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction).
- [X] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md
Comment (PiousSutherland) [inline books/free-programming-books-langs.md:1]: This:
```
* [Software Architecture with Spring 5.0](https://www.packtpub.com/free-ebooks/software-architecture-spring-50) - René Enríquez, Alberto Salazar (Packt account *required*)
* [Spring Framework Cookbook: Hot Recipes for Spring Framework](https://www.javacodegeeks.com/wp-content/uploads/2017/01/Spring-Framework-Cookbook.pdf) - JCGs (Java Code Geeks) (PDF)
* [Spring Framework Notes for Professionals](https://goalkicker.com/SpringFrameworkBook) - Compiled from StackOverflow documentation (PDF)
* [Spring Framework Reference Documentation](https://docs.spring.io/spring/docs/current/spring-framework-reference/) - Rod Johnson, et al.
* [Spring books and guides](https://www.baeldung.com/baeldung-ebooks) - Baeldung (PDF sent by email)
```

Needs to change to this (alphabtical):
```
* [Software Architecture with Spring 5.0](https://www.packtpub.com/free-ebooks/software-architecture-spring-50) - René Enríquez, Alberto Salazar (Packt account *required*)
* [Spring books and guides](https://www.baeldung.com/baeldung-ebooks) - Baeldung (PDF sent by email)
* [Spring Framework Cookbook: Hot Recipes for Spring Framework](https://www.javacodegeeks.com/wp-content/uploads/2017/01/Spring-Framework-Cookbook.pdf) - JCGs (Java Code Geeks) (PDF)
* [Spring Framework Notes for Professionals](https://goalkicker.com/SpringFrameworkBook) - Compiled from StackOverflow documentation (PDF)
* [Spring Framework Reference Documentation](https://docs.spring.io/spring/docs/current/spring-framework-reference/) - Rod Johnson, et al.
```

File: courses/free-courses-fr.md
Comment (PiousSutherland) [inline courses/free-courses-fr.md:1]: This:
```
* [C# - Cours](https://www.youtube.com/playlist?list=PLrSOXFDHBtfGBHAMEg9Om9nF_7R7h5mO7) - Formation Video
* [Apprendre le C#](https://www.youtube.com/playlist?list=PLkHw7J3J2iapWFUnQmVzsRCU5YxaAWHSY) - Pentiminax
```

Changes to:
```
* [Apprendre le C#](https://www.youtube.com/playlist?list=PLkHw7J3J2iapWFUnQmVzsRCU5YxaAWHSY) - Pentiminax
* [C# - Cours](https://www.youtube.com/playlist?list=PLrSOXFDHBtfGBHAMEg9Om9nF_7R7h5mO7) - Formation Video
```

**Review by PiousSutherland — CHANGES_REQUESTED** [review]

File: courses/free-courses-fr.md
Comment (C-Alexis4414) [reply courses/free-courses-fr.md:1]: I corrected the order in the French courses for references in C# and I removed the reference to the books on Spring.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**